### PR TITLE
Replace launcher.py by a Makefile

### PR DESCRIPTION
**Pull request by BenoitRoux0**
I've replaced the launcher.py by a Makefile with a rule to update the repo and another to run main.py.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PYTHON := ${shell whereis -b python | cut -d " " -f2}
```
Comment (BenoitRoux0) [inline Makefile:1]: I use the whereis command. After I cut the output, to get the first Python executable found.

**Comment by tdameros** [comment]
This is a good idea but for the update with git pull it is necessary to specify the remote and the branch

git pull origin main

Moreover, on Linux the command python3 is used instead of python.

**Comment by BenoitRoux0** [comment]
> Moreover, on Linux the command python3 is used instead of python.


The whereis python command finds also python3.

**Comment by tdameros** [comment]
No, look

Python error :
python

Make update error :
makeerror

**Pull request closed without merging**